Installation/Mapping/buren.pub.typeSourceInBody.mapping.json

```diff
@@ -2,10 +2,10 @@
   "title": "pub.typeSourceInBody",
   "$id": "https://buren.nl/mapping/buren.pub.typeSourceInBody.mapping.json",
   "$schema": "https://docs.commongateway.nl/schemas/Mapping.schema.json",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "passTrough": false,
   "mapping": {
-    "#": "[{% for object in # %}{% if not loop.first %},{% endif %}{{ map('https://buren.nl/mappings/openPubType.mapping.json', object)|json_encode }}{% endfor %}]"
+    "#": "[{% for key, object in _context %}{% if key matches '/^\\\\d+$/' %}{% if not loop.first %},{% endif %}{{ map('https://buren.nl/mapping/buren.pub.typeSourceInBodyItem.mapping.json', object)|json_encode }}{% endif %}{% endfor %}]"
```

Comment:
Fine for now, but we should come up with a better solution for _context mappings...

Reply:
Yeah, we should change it in a way that we always put everything we need for mapping in a _context.body parameter or something like that

Reply:
But keep backwards compatibility. I'm already a fan 😄